frontend/src/modules/Survey/Components/StepBegin.tsx

```diff
@@ -40,53 +47,75 @@ export const StepBegin = ({
-        </StyledTitleWrapper>
-        <StyledFields>
-          <TextField
-            placeholder="Name"
-            variant="standard"
-            sx={textInputStyle}
-            value={name}
-            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-              setName(e.target.value)
-            }
-          />
-          <TextField
-            value={email}
-            sx={textInputStyle}
-            variant="standard"
-            type="email"
-            onBlur={() => setIsValidEmail(validEmail.test(email))}
-            placeholder="Cornell Email"
-            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-              setEmail(e.target.value)
-            }
-            error={!isValidEmail}
-            helperText={isValidEmail ? ' ' : 'Invalid Email'}
-          />
-        </StyledFields>
-        <Box
-          sx={{
-            marginLeft: 'auto',
-            textAlign: 'center',
-            color: 'purple.100',
-            weight: 600,
-          }}
-        >
-          <IconButton
-            className="next"
-            onClick={gotoNextStep}
-            disabled={name === '' || email === '' || !isValidEmail}
-            sx={{ boxShadow: 3 }}
+        <main>
+          <StyledTitleWrapper>
+            <StyledHeaderText>Hi,</StyledHeaderText>
+            <StyledWelcomeText>Find study partners!</StyledWelcomeText>
+          </StyledTitleWrapper>
+
+          <StyledFields>
+            <InputLabel htmlFor="user name">
+              {' '}
+              <StyledLabelText> Name: </StyledLabelText>{' '}
+            </InputLabel>
+            <TextField
+              id="user name"
+              variant="standard"
+              aria-label="user name"
+              placeholder="Martha E. Pollack"
+              // variant="standard"
+              sx={textInputStyle}
+              value={name}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                setName(e.target.value)
+              }
+            />
+            <InputLabel htmlFor="user email">
+              {' '}
+              <StyledLabelText> Email: </StyledLabelText>{' '}
+            </InputLabel>
+            <TextField
+              id="user email"
+              variant="standard"
+              aria-label="user email"
+              value={email}
+              sx={textInputStyle}
+              type="email"
+              onBlur={() => setIsValidEmail(validEmail.test(email))}
+              placeholder="mep22@cornell.edu"
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                setEmail(e.target.value)
+              }
+              error={!isValidEmail}
+            />
+            <FormHelperText id="email-helper-text" sx={{ fontColor: 'red' }}>
+              {' '}
+              <StyledErrorText>
+                {isValidEmail ? ' ' : 'Invalid Email'}
+              </StyledErrorText>
+            </FormHelperText>
```

Comment:
What is wrong with the existing helper text that is part of MUI? May be able to style it if needed. Also `fontColor: red` is not constrasty enough

Reply:
https://mui.com/material-ui/react-text-field/#accessibility 

This is what I referenced and if I use the helperText prop of the TextFeild component, I end up with a broken aria label error, so I did it this way.

Reply:
Hm, I think FormHelperText is already part of the TextField and what you might be looking at in the documentation is for "If you are composing the component" which we are not

Reply:
I think this is because the id of the TextField was `"user email"`, with the space. Removing the space fixed it and I was able to get the broken aria label working.